internal/impl/azure/auth.go

```diff
@@ -122,16 +108,48 @@ func getEmulatorConnectionString(blobServicePort, queueServicePort, tableService
 	)
 }
 
+const (
+	azuriteBlobPortEnv  = "AZURITE_BLOB_ENDPOINT_PORT"
+	azuriteQueuePortEnv = "AZURITE_QUEUE_ENDPOINT_PORT"
+	azuriteTablePortEnv = "AZURITE_TABLE_ENDPOINT_PORT"
+)
+
+func parseStorageConnectionString(storageConnectionString, storageAccount string) string {
```

Comment:
The name `parseStorageConnectionString` doesn't seem to fit here, since this function is used for all three services. I'd consider using the following signature instead: `func getConnectionString(connectionString, account string) string`. On the happy path, it returns the input `connectionString` as-is.